workshops/facial_expression_recognizer/README.md

```diff
@@ -0,0 +1,155 @@
+  * Next we will assign `FER()` which was imported earlier to the **detector** variable.
+  * print syntax with `detector.detect_emotions(image)` as parameter will returns dictionary of bounding box notations.
+Below is Sample Output:
+```python
+[{'box': [277, 90, 48, 63], 'emotions': {'angry': 0.02, 'disgust': 0.0, 'fear': 0.05, 'happy': 0.16, 'neutral': 0.09, 'sad': 0.27, 'surprise': 0.41}]
+```
+**NOTE** If you want to add an image in `repl.it` IDE refer to the below Video:
+
+![Upload Image Guide](https://cloud-3jh0ilrgt.vercel.app/0uploadimage.gif)
+
+## What is going on underhood🤔🤔
+
+The `detector` is the main part of our workshop. Let's see what the detector do:
+A detector is an object created to store `FER()`. Detector returns an Ordered Dictionary of Bounding Box Notations. It observes and detects where the face is situated, classifies, and rates its probability with decimal values from `0` to `1` for all 6 emotions respectively. FER exhibit Keras model built with CNN or well know as Convolutional Neural Networks. It stores the values in the `HDF5` model. FER by default detect faces using OpenCV's  Haar Cascade Classifier. Alternatively, we can use a more advanced Multi-Cascade Convolutional Network in short `MTCNN`. It will use Peltarion API in the backend in place of the Keras model. By default FER uses CNN but if want to use advanced MTCNN just pass `mtcnn=True` in FER parentheses just like `detector = FER(mtcnn=True)`. for further use we will use the MTCNN model for more accurate results.
+
+## Data is the new oil!
+
+Data is known as the new oil of the Twenty-First Century. We can do different things with data generated above which we will discuss in the `Hacking Section` at last so stay tuned. We will store the emotion and detected face. Then display bounding yellow box around face detected and scores below it. Let's see below how we can do this.
+```python 
+import cv2
+from fer import FER
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+
+image = cv2.imread("abc.jpg")
+detector = FER(mtcnn=True)
+```
+Until now we have reached till here.
+
+Next, we will declare a new variable `result` and store our output in form of a separate array.
+```python
+result = detector.detect_emotions(image)
+```
+### Now it is time to Highlight Yellow Bounding Box around face:
+```python
+bounding_box = result[0]["box"]
+emotions = result[0]["emotions"]
+cv2.rectangle(image,(bounding_box[0], bounding_box[1]),(bounding_box[0] + bounding_box[2], bounding_box[1] + bounding_box[3]),(0, 155, 255), 2,)
+```
+Explanation of the above code is:
+  * `bounding_box` variable will store bounding box coordinates that were generated and stored in the `result` variable. It will access and start storing from its `Zeroth`elemets (or the first element in common language) of `" box"` key. If you print this variable you will get output something like this `(298, 361, 825, 825)`.
```

Comment:
```suggestion
  * `bounding_box` variable will store the bounding box coordinates that were generated and stored in the `result` variable. It will start storing from its `0th`element (or the first element in common language) of the `"box"` key. If you print this variable you will get an output something like this `(298, 361, 825, 825)`.
```